contracts/DAIInterestRateModelV2.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity ^0.5.16;
+
+import "./JumpRateModel.sol";
+import "./SafeMath.sol";
+
+/**
+  * @title Compound's DAIInterestRateModel Contract (version 2)
+  * @author Compound
+  * @notice The parameterized model described in section 2.4 of the original Compound Protocol whitepaper.
+  * Version 2 modifies the original interest rate model by increasing the "gap" or slope of the model prior
+  * to the "kink" from 0.05% to 2% with the goal of "smoothing out" interest rate changes as the utilization
+  * rate increases.
+  */
+contract DAIInterestRateModelV2 is JumpRateModel {
+    using SafeMath for uint256;
+
+    /**
+     * @notice The additional margin per block separating the base borrow rate from the roof (2% / block).
+     * Note that this value has been increased from the original value of 0.05% per block.
+     */
+    uint256 public constant gapPerBlock = 2e16 / blocksPerYear;
+
+    /**
+     * @notice The assumed (1 - reserve factor) used to calculate the minimum borrow rate (reserve factor = 0.05)
+     */
+    uint256 public constant assumedOneMinusReserveFactorMantissa = 0.95e18;
+
+    PotLike pot;
+    JugLike jug;
+
+    /**
+     * @notice Construct an interest rate model
+     * @param jumpMultiplierPerYear The multiplierPerBlock after hitting a specified utilization point
+     * @param kink_ The utilization point at which the jump multiplier is applied
+     * @param pot_ The address of the Dai pot (where DSR is earned)
+     * @param jug_ The address of the Dai jug (where SF is kept)
+     */
+    constructor(uint256 jumpMultiplierPerYear, uint256 kink_, address pot_, address jug_) JumpRateModel(0, 0, jumpMultiplierPerYear, kink_) public {
+        pot = PotLike(pot_);
+        jug = JugLike(jug_);
+        poke();
+    }
+
+    /**
+     * @notice Calculates the current supply interest rate per block including the Dai savings rate
+     * @param cash The total amount of cash the market has
+     * @param borrows The total amount of borrows the market has outstanding
+     * @param reserves The total amnount of reserves the market has
+     * @param reserveFactorMantissa The current reserve factor the market has
+     * @return The supply rate per block (as a percentage, and scaled by 1e18)
+     */
+    function getSupplyRate(uint256 cash, uint256 borrows, uint256 reserves, uint256 reserveFactorMantissa) public view returns (uint256 supplyRateMantissa) {
```

Comment:
Not a huge fan of all the stylistic changes, I think the bike-shedding could be worse on-chain where the next change someone wants to make moves to a different style, and all the contracts in the protocol have different conventions and huge diffs between them